test-alerts.cpp

```diff
@@ -3,6 +3,19 @@
 #include "test/catch.hpp"
 #include "typewise-alert.h"
 
-TEST_CASE("infers the breach according to limits") {
-  REQUIRE(inferBreach(12, 20, 30) == TOO_LOW);
+
+TEST_CASE("check and alert through email") {
+	alertToEMAIL* emailmsgobj = new alertToEMAIL;
+	BatteryCharacter batteryObj = { HI_ACTIVE_COOLING , "Exide" };
+	REQUIRE(checkAndAlert(emailmsgobj, batteryObj, -5) == TOO_LOW);
```

Comment:
This REQUIRE statement conveys that checkAndAlert converts the parameters to one return value (low/high)
If you had to re-state its functionality as data-conversion, what would you state?